lib/std/c.zig

```diff
@@ -5704,6 +5704,21 @@ pub const MSG = switch (native_os) {
         pub const WAITFORONE = 0x2000;
         pub const NOTIFICATION = 0x4000;
     },
+    .openbsd => struct {
```

Comment:
Please add in a comment a permalink to the relevant lines of the OpenBSD header that these come from.

Reply:
@knowmercy pointing to [current latest version of `sys/socket.h`](https://github.com/openbsd/src/blob/42a7be81bef70c04732f45ec573622effe56b563/sys/sys/socket.h#L506) should be fine